06-classification_continued.Rmd

```diff
@@ -479,7 +576,8 @@ new data: if we had a different training set, the predictions would be
 completely different.  In general, if the model *is influenced too much* by the
 training data, it is said to **overfit** the data.
```

Comment:
Should we mention that such a classifier will not generalize well to new data?

Reply:
says that in the previous sentence